scripts/compare_signatures.py

```diff
@@ -0,0 +1,98 @@
+import re
```

Comment:
Can change the output to print documentation and doxygen in the same order?

From
```
Checking function signatures for 
  Documentation file:  www/Magick++/Pixels.html 
  Doxygen file:  www/api/Magick++/classMagick_1_1Pixels.html 

- Return type mismatch for method "x"
Doxygen return type   : ssize_t
Documentation returns : ['int'] 
```

To
```
Checking function signatures for 
  Doxygen file       : www/api/Magick++/classMagick_1_1Pixels.html 
  Documentation file : www/Magick++/Pixels.html 

- Return type mismatch for method "x"
Doxygen return type   : ssize_t
Documentation returns : ['int'] 
```